Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Confirm.hs

```diff
@@ -112,6 +119,10 @@ handler transporter req quote = do
       case quote of
         Left (driver, driverQuote) -> do
           ride <- buildRide driver.id booking
+          version <- asks (.version)
```

Comment:
Create a separate file inside Tools folder which will contains all functions related to eventStream. We should be just calling the function from here, all payload creation and event creation should be inside that function and then that function should be triggering the event.

Reply:
Yeah, agree, also in Tools module you can use our domain types instead of Text: Id Person, Id Merchant, Id Ride RideStatus etc

Reply:
Done